recipes/redmapper/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "redmapper" %}
+{% set version = "0.7.3" %}
+{% set sha256 = "069a8702c706c4d1695b7170d0cc1eeb72f24c913a985504b54916270027199e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/erykoff/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<37 or win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - gsl
+    - libcblas
+  run:
+    - python
+    - astropy
+    - matplotlib
+    - pyyaml
+    - fitsio
+    - esutil
+    - numpy
+    - healpy
+    - scipy
+    - healsparse
+    - gsl
```

Comment:
gsl has a run export so no need for it here in run

see https://github.com/conda-forge/gsl-feedstock/blob/master/recipe/meta.yaml#L20